experiments/dataSources/Github/readme.md

```diff
@@ -0,0 +1,55 @@
+# Connect to Github.com APIs via a Skuid REST Data Source 
+
+Many companies leverage Github for source control and as a secure source code repository. For any organization in a regulated industry, this brings compliance obligations including auditability of access control, authorization, and change management. This guide is a tutorial for creating an auditing dashboard leveraging Skuid and Github to help you meet these obligations. 
+
+# Setting up your Github service account and API token 
```

Comment:
Time for the copy suggestions! Get ready for the nitpickiest of nitpicks.

Let's make sure that the rest of these headers are in a proper hierarchy. We should typically only have one H1 in these READMEs.
```suggestion
## Setting up your Github service account and API token 
```